src/main/java/uk/nhs/digital/nhsconnect/lab/results/configuration/AmqpAutoConfiguration.java

```diff
@@ -0,0 +1,22 @@
+package uk.nhs.digital.nhsconnect.lab.results.configuration;
+
+import org.apache.qpid.jms.JmsConnectionFactory;
+import org.springframework.boot.autoconfigure.AutoConfigureAfter;
+import org.springframework.boot.autoconfigure.AutoConfigureBefore;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.boot.autoconfigure.jms.JmsAutoConfiguration;
+import org.springframework.boot.autoconfigure.jms.JndiConnectionFactoryAutoConfiguration;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+import javax.jms.ConnectionFactory;
+
+@Configuration
+@AutoConfigureBefore(JmsAutoConfiguration.class)
+@AutoConfigureAfter({JndiConnectionFactoryAutoConfiguration.class})
+@ConditionalOnClass({ConnectionFactory.class, JmsConnectionFactory.class})
+@ConditionalOnMissingBean(ConnectionFactory.class)
+@Import({AmqpConfiguration.class})
+public class AmqpAutoConfiguration {
```

Comment:
Is this class being used anywhere? If not, then maybe it should be deleted?

Reply:
We can, yes. e4531916679c1b294b2db95232e49082bd43ce7d Thanks!